image-mirror.sh

```diff
@@ -3,7 +3,7 @@
 set -uo pipefail
 export DOCKER_CLI_EXPERIMENTAL="enabled"
 
-ARCH_LIST="amd64 arm64 arm"
+ARCH_LIST="amd64 arm64 arm s390x"
```

Comment:
Is there a image we are targeting with this ? How would we know if it is working.

Reply:
For example, it'll be useful for https://hub.docker.com/r/flannelcni/flannel-cni-plugin/tags. Currently, the s390x arch is not being mirrored https://hub.docker.com/r/rancher/mirrored-flannelcni-flannel-cni-plugin/tags